misc/testerina/modules/testerina-core/src/main/java/org/ballerinalang/testerina/core/BTestRunner.java

```diff
@@ -622,7 +614,7 @@ private void execute(boolean buildWithTests) {
         outStream.println("Running tests");
         // Check if the test suite is empty
         if (testSuites.isEmpty()) {
-            outStream.println("    No tests found");
+            outStream.println("\t" + "No tests found");
```

Comment:
I am not sure, using "\t" in CLI is correct. @kaviththiranga @hevayo @NipunaMarcus Could you please comment. ?

Reply:
we use \t in other places to indent so this should be fine for consistency. But agree to use spaces instead of /t.